src/components/ProblemsTable.tsx

```diff
@@ -244,6 +283,27 @@ export const ProblemsTable = ({
 
   return (
     <Table
+      onRowClick={(event: React.MouseEvent<Element, MouseEvent>, row: Row<TableRow | TableRow['subRows']>) => {
+        const element = event.target as HTMLTableRowElement;
+        element.parentElement?.classList.add('active');
+        context?.setCurrentAuditInfo({
+          ...row.original,
+          fileName: row.original.filename ?? getFileNameFromId(row.original.fileId),
+        });
+
+        // event listener added as component does not have an 'outside clicked' handler
+        // and cannot wrap individual row in a custom react hook
+        document.addEventListener(
```

Comment:
can't we use useState and useEffect for a row expansion? You could keep the row state, currently active row in states and open, close it with useEffect?